apps/ui/src/components/views/project-settings-view/orphaned-features-section.tsx

```
@@ -0,0 +1,653 @@
+          featureIds,
+          action,
+          targetBranch
+        );
+        if (result.success || (result.resolvedCount && result.resolvedCount > 0)) {
+          const resolvedIds = new Set(
+            result.results?.filter((r) => r.success).map((r) => r.featureId) ?? featureIds
+          );
+          setOrphanedFeatures((prev) => prev.filter((o) => !resolvedIds.has(o.feature.id)));
+          setSelectedIds((prev) => {
+            const next = new Set(prev);
+            resolvedIds.forEach((id) => next.delete(id));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)